api/types/image.go

```diff
@@ -12,10 +12,15 @@ type Image struct {
 	// NewName is the value used to replace the original name.
 	NewName string `json:"newName,omitempty" yaml:"newName,omitempty"`
 
+	// TagSuffix is the value used to suffix the original tag
+	// If digest or newtag is present TagSuffix value is ignored
```

Comment:
Would it be better to throw an error in the case that digest or newtag is present when a tagSuffix is specified, rather than silently do nothing?

Reply:
in the if condition added a nested condition to check for such a case and throw error if needed